sale_quotation/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module handle the model and view for product and sale.order needs for shopinvader-api-quotation
```

Comment:
You should define the context and the 'why' of this addon. This description doesn't provide the minimal information to know when to use it.